### ISSUE TITLE
Dependency not available when trying to install a modpack

### ISSUE THREAD

**Issue by DomHam9** [KSP-CKAN/CKAN#3225]
Background
----------
* Operating System: Windows 10 64 bit
* CKAN Version: v1.29.2
* KSP Version: 1.10.1.2939

**Have you made any manual changes to your GameData folder (i.e., not via CKAN)?**
No

Problem
-------
**Describe the bug**
I am trying to install a mod list via a .ckan file which contains the mod at hand.
I have even downloaded the exact mod and put into my gamedata and it still has issues

I am trying to install a mod list from a .ckan file and it has a mod within it yet it says that it it still requires it but it is not listed in the index, or not available for your version of ksp

**Steps to reproduce**
* Go to...   https://www.dropbox.com/s/qdlrtllnhynogy1/ComingHome-Updated.ckan?dl=0
* Click on...  download the .ckan file and open it with ckan
* See error

**Expected behavior**

**Screenshots (if applicable)**
![Throughtheeyesofakerbal](https://user-images.githubusercontent.com/75277103/101259121-d8285300-371e-11eb-9d87-7175a3d3d2da.PNG)

**CKAN error code (if applicable):**

```
PLACE CODE HERE !
```

**Comment by HebaruSan** [comment]
ThroughTheEyesOfaKerbal is only marked as compatible up to KSP 1.8.

**Comment by DomHam9** [comment]
to fix it could i remove the mod from the ckan listing?

**Comment by HebaruSan** [comment]
You could do that, or you could make your CKAN version requirements more permissive:
https://github.com/KSP-CKAN/CKAN/wiki/User-guide#choosing-compatible-game-versions

**Comment by DomHam9** [comment]
Ok i will give that a go

**Comment by HebaruSan** [comment]
Sounds good, please let us know how it turns out.

**Comment by DomHam9** [comment]
well other than it making me choose between mods that share the same mudules, i managed to open into the game. However, the first time I managed to load into the game after deleting excess mods that I don't need and were preventing the game from starting, I clicked on the VAB and the game came up with the wait for game or close. I then reloaded the game and upon loading into the save, i got a unity crash and it made content on my left monitor (the one the game is on) glitch and drag like old windows versions would get, one time both of my monitors turned black and my friend said that Unity had prevented my gpu from communicating with my pc haha. In short, I might have to remove a few more mods - but thanks for your help in downloading the mods 👍
